src/manifest.firefox.json

```diff
@@ -2,13 +2,19 @@
   "name": "Toggl Plan: Project Planning Calendar",
   "version": "2.0.4",
   "manifest_version": 2,
+  "author": "Toggl <support@toggl.com>",
```

Comment:
Looks like this is really the only addition needed :-D

Reply:
> Looks like this is really the only addition needed :-D

How about we discard this branch altogether and just add this in a new branch? Just to be sure that everything works fine.